### PR TITLE
extract event store schema into own class to make schema creation easier

### PR DESCRIPTION
**Pull request by schnipseljagd**

### CODE REVIEW

File: tests/LiteCQRS/Plugin/Doctrine/TableEventStoreTest.php [hunk 1]
```diff
@@ -4,7 +4,8 @@
 
 use Doctrine\DBAL\DriverManager;
 use Doctrine\DBAL\Schema\Schema;
-use LiteCQRS\Plugin\Doctrine\EventStore\TableEventStore;
+use LiteCQRS\Plugin\Doctrine\EventStore\TableEventStore,
+    LiteCQRS\Plugin\Doctrine\EventStore\TableEventStoreSchema;
```
Comment (stof) [inline tests/LiteCQRS/Plugin/Doctrine/TableEventStoreTest.php]: you should use a `use` statement per class to be consistent

File: src/LiteCQRS/Plugin/Doctrine/EventStore/TableEventStore.php [hunk 2]
```diff
@@ -18,7 +18,7 @@ class TableEventStore implements EventStoreInterface
     private $table;
     private $serializer;
 
-    public function __construct(Connection $conn, SerializerInterface $serializer, $table = 'litecqrs_events')
```
Comment (beberlei) [inline src/LiteCQRS/Plugin/Doctrine/EventStore/TableEventStore.php:21]: Why did you remove it here? Shouldnt it still be the default?

Comment (schnipseljagd) [inline src/LiteCQRS/Plugin/Doctrine/EventStore/TableEventStore.php:21]: Because it is the default in the schema class too. So i removed it here.

Comment (stof) [inline src/LiteCQRS/Plugin/Doctrine/EventStore/TableEventStore.php:21]: but it means that creating the TableEventStore now requires to pass the table name explicitly, so you don't have a default anymore

Comment (schnipseljagd) [inline src/LiteCQRS/Plugin/Doctrine/EventStore/TableEventStore.php:21]: Yep.

File: composer.json [hunk 3]
```diff
@@ -4,14 +4,5 @@
     "license": "MIT",
     "autoload": {
         "psr-0": {"LiteCQRS": "src/"}
-    },
```
Comment (beberlei) [inline composer.json:7]: can you change that back?

**Comment by henrikbjorn** [comment]
What is the status of this? Is it just to readd the default schema name?

**Comment by beberlei** [comment]
Its not mergable, because of the composer changes. But i would merge it.

**Comment by schnipseljagd** [comment]
It's ok now?

**Comment by henrikbjorn** [comment]
@schnipseljagd you still need to readd the optional $table argument in TableEventStore as it was before.

**Comment by schnipseljagd** [comment]
ok, yeah so i have it now ;-)

**Comment by henrikbjorn** [comment]
There is one problem with this, if it is a composite id it will json encode it, but that is not really that good to query on and the value will be different from different aggregates.

**Comment by beberlei** [comment]
Yes, i will remove that part. Composite keys require a custom storage that the user has to code himself.

**Comment by beberlei** [comment]
done in b3c840f7b271dd0e41dd9b81824e1a313c9fbfe8
